code/modules/requisitions/supply_export.dm

```diff
@@ -33,7 +37,120 @@
 		if(JOB_CAT_COMMAND)
 			. = 100
 	SSpoints.supply_points[faction_selling] += .
-	return new /datum/export_report(., name, faction_selling)
+	SSpoints.export_history += new /datum/export_report(., name, faction_selling)
+
+/**Calculates return value of properly prepared corpses, handles point payout and export logging.
+ *
+ * Checks for:
+ **Five minute defib timer
+ **Any brute or burn damage, including internal bleeding
+ **Any limbs requiring repair, bearing an open surgical incision, or carrying high germs
+ **Eye damage or disfigurement
+ **Missing synth head, on top of the normal missing limb penalty
+ **Shrapnel
+ **Full dress uniform: torso, hat, gloves, boots
+ **Suicide/export before shutters open
+ **Dogtag in memorial, if they've got one
+ */
+/mob/living/carbon/human/proc/burial_export(faction_selling)
+	var/report_name = name
+	var/list/problems = list()
+	var/payout = 0
+
+	switch(job.job_category)
+		if(JOB_CAT_ENGINEERING || JOB_CAT_MEDICAL || JOB_CAT_REQUISITIONS)
+			payout = 5	//Shipside crew shouldn't be dying in any situation where you can still use requisitions.
+		if(JOB_CAT_MARINE)
+			payout = 20
+		if(JOB_CAT_SILICON)
+			payout = 50	//Silicons can get recycled so they're worth extra
+		if(JOB_CAT_COMMAND)
+			payout = 30
+
+	if(!HAS_TRAIT(src, TRAIT_UNDEFIBBABLE))
+		problems += "lingering vitals"
+		payout = 0
+
+	var/damaged_limbs = 0
+	var/surgery_limbs = 0
+	var/infected_limbs = 0
+	var/tissue_damage = 0
+	for(var/datum/limb/test_limb AS in limbs)
+		if(test_limb.limb_status & (LIMB_BLEEDING|LIMB_BROKEN|LIMB_DESTROYED|LIMB_NECROTIZED|LIMB_AMPUTATED|LIMB_MUTATED))
+			damaged_limbs++
+		if(test_limb.surgery_open_stage)
+			surgery_limbs++
+		if(test_limb.germ_level > INFECTION_LEVEL_ONE)
+			infected_limbs++
+		tissue_damage += test_limb.brute_dam + test_limb.burn_dam
+	if(damaged_limbs)
+		payout -= 3 * damaged_limbs
+		problems += "damaged limbs"
+	if(surgery_limbs)
+		payout -= 5 * surgery_limbs
+		problems += "ongoing surgery"
+	if(infected_limbs)
+		payout -= 3 * infected_limbs
+		problems += "infection"
+	if(tissue_damage > 10)
+		payout -= round(tissue_damage/10)
+		problems += "tissue damage"
```

Comment:
i don't really get the point of those, you want someone to do the coroner?

Cause either you can't do things about this, and i don't think it's really fair to give them more, or you can fix the issues but probably no one will bother to

Reply:
Yeah, the point is to put effort in shipside to get them fixed up. Part of the payout balance is on the idea that a: people won't bother and b: when they do bother, they'll probably miss something here or there.